features/step_definitions/auditing_steps.rb

```diff
@@ -3,15 +3,19 @@
 Then("there is an audit entry containing {string}") do |content|
   step "I visit the audit log"
 
-  within_table("Activity log") do
+  within_table("Audit log") do
     expect(page).to have_content content
   end
 end
 
+When("I click on the audit accordion") do
+  page.find("#audit-log").click
+end
+
 When("I visit the audit log") do
   steps %(
     When I view the planning application
-    Then I press "Key application dates"
-    Then I press "Activity log"
+    Then I click on the audit accordion
```

Comment:
why not keep `Then I press "Audit log"`?

Reply:
it can't find it, that's why I added the other "page.find(#audit-log) since with just "press "Audit log" it gives:
```
    And I cancel a validation request for a "Meme of the dog" with "Meme of dog is no longer needed" # features/step_definitions/validation_requests_steps.rb:102
    When I view the planning application audit                                                       # features/step_definitions/planning_application_steps.rb:105
    Then there is an audit entry containing "Cancelled: validation request (new document#1)"         # features/step_definitions/auditing_steps.rb:3
      Unable to find link or button "Activity log" (Capybara::ElementNotFound)
      ./features/step_definitions/web_steps.rb:71:in `nil'
      ./features/step_definitions/auditing_steps.rb:16:in `nil'
      ./features/step_definitions/auditing_steps.rb:4:in `nil'
      features/auditing.feature:26:in `there is an audit entry containing "Cancelled: validation request (new document#1)"'
      ```

Reply:
because it can't find it, so I added the id finder just above that:
```
    Then the page contains "Description change request successfully cancelled"     # features/step_definitions/web_steps.rb:3
    And there is an audit entry containing "Cancelled: description change request" # features/step_definitions/auditing_steps.rb:3
      Unable to find link or button "Audit log" (Capybara::ElementNotFound)
      ./features/step_definitions/web_steps.rb:71:in `nil'
      ./features/step_definitions/auditing_steps.rb:16:in `nil'
      ./features/step_definitions/auditing_steps.rb:4:in `nil'
      features/adding_description_change.feature:16:in `there is an audit entry containing "Cancelled: description change request"'
      ```

Reply:
OK I've had a look, it's because GOV.UK is inconsistent in their presentation of the accordion component. The markup on the component page says the heading text is a span, but the actual HTML example uses a button: https://design-system.service.gov.uk/components/accordion/default/index.html. Let's use a button and remove the ID match.